mvlearn/embed/kcca.py

```diff
@@ -3,29 +3,64 @@
 ====================================
 Python module for kernel canonical correlation analysis (kCCA)
 
-Code modified from UC Berkeley, Gallant lab
-(https://github.com/gallantlab/pyrcca)
-Copyright 2016, UC Berkeley, Gallant lab.
+Adopted from Steven Van Vaerenbergh's MATLAB Package 'KMBOX'
+https://github.com/steven2358/kmbox
 """
 
```

Comment:
Can you copy the license from here https://github.com/neurodata/mvlearn/blob/master/mvlearn/embed/gcca.py and put it at the top of the file instead of the current top 4 lines you have? Just put the "Adopted..." and github link lines in the same comment format, like 

```mvlearn
# Copyright 2019 NeuroData (http://neurodata.io)
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
# Adopted from Steven Van Vaerenbergh's MATLAB Package 'KMBOX'
# https://github.com/steven2358/kmbox
```